service/src/main/kotlin/fi/espoo/evaka/shared/security/Action.kt

```diff
@@ -437,6 +437,20 @@ sealed interface Action {
             READ_ATTENDANCE_SUMMARY(
                 IsCitizen(allowWeakLogin = true).guardianOfChild(),
                 IsCitizen(allowWeakLogin = true).fosterParentOfChild()
+            ),
+            READ_CHILD_DOCUMENTS(
+                IsCitizen(allowWeakLogin = true).guardianOfChild(),
```

Comment:
Is weak login checked to be enough for pedagogic docs?

Reply:
😱 probably not, thanks!

Reply:
fixed